src/wizeng.main.v3

```diff
@@ -283,6 +284,7 @@ component WizengOptions {
 	def group = OptionsRegistry.newGroup("WIZENG", null);
 	def HELP = group.newFlag("help", printHelp, "Print command-line help.");
 	def HELP2 = group.newFlag("-help", printHelp, "Print command-line help.");
+	def PROG_ARGS = group.newOption("-args", null, "=<arg_1, ... ,arg_n>", parseArgs, "Provide a list of program args for certain host module initialization.");
```

Comment:
programArgs used to be optional args appended after the single wasm file is provided. Now that we are accepting multiple wasm files, I made programArgs an explicit option to avoid mistaking it for wasm files

Reply:
I think we should keep the behavior that arguments after the `.wasm` files are arguments to the program; this is standard practice. If there is ambiguity between the end of the list of `.wasm` files and the start of the arguments to the (main) module, then most engines (including JS engines) use `--` to slit the "program" from its arguments. Let's keep the program args behavior for now and implement the `--` behavior in the next PR.